### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by wvega**
## Summary

Base branch for v2.0.0

### Stories

- [CH 54003](https://app.clubhouse.io/skyverge/story/54003)

## Before merge

- [x] Test suites pass
- [x] Confirm the changes introduced in this PR work in each supported minor WooCommerce version

## Deploy

- Use `npx sake deploy --deployAssets=0` to skip the `shell:svn_commit_assets` action

### CODE REVIEW

File: includes/Handlers/Connection.php [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+	 */
+	public function get_external_business_id() {
+
+		if ( ! is_string( $this->external_business_id ) ) {
+
+			$value = get_option( self::OPTION_EXTERNAL_BUSINESS_ID );
+
+			if ( ! is_string( $value ) ) {
+
+				$value = sanitize_title( get_bloginfo( 'name' ) ) . '-' . uniqid();
```
Comment (unfulvio) [inline includes/Handlers/Connection.php:169]:
```suggestion
				$value = sanitize_title( get_bloginfo( 'name' ) ) . '-' . uniqid( '', true );
```

Comment (unfulvio) [reply includes/Handlers/Connection.php:169]: not sure if uniqid here should have 2nd param as true

Comment (wvega) [reply includes/Handlers/Connection.php:169]: I'm not sure if the `.` that is added to the unique ID is allowed. What about:
```suggestion
				$value = sanitize_title( uniqid( get_bloginfo( 'name' ) . '-', true ) )
```

That should give something like:

    skyverge-plugins-5ebbeab9843e86-78530995

**Comment by wvega** [comment]
@cfloody [Facebook for WooCommerce 2.0.0](https://wordpress.org/plugins/facebook-for-woocommerce/) is out now and I saw that you [imported](https://github.com/facebookincubator/facebook-for-woocommerce/pull/1260#issuecomment-622083917) the PR for one of the previous releases. Would you be able to merge this PR into the main branch too?

**Comment by cfloody** [comment]
> @cfloody [Facebook for WooCommerce 2.0.0](https://wordpress.org/plugins/facebook-for-woocommerce/) is out now and I saw that you [imported](https://github.com/facebookincubator/facebook-for-woocommerce/pull/1260#issuecomment-622083917) the PR for one of the previous releases. Would you be able to merge this PR into the main branch too?

@wvega Yes no problem, I will perform the merge on Monday

**Pull request closed without merging**